monai/handlers/utils.py

```diff
@@ -59,6 +59,7 @@ def write_metrics_reports(
     metrics: dict[str, torch.Tensor | np.ndarray] | None,
     metric_details: dict[str, torch.Tensor | np.ndarray] | None,
     summary_ops: str | Sequence[str] | None,
+    class_labels: list[str] | None = None,
```

Comment:
This will  have to be made the last argument in the constructor otherwise any invocation using only positional arguments will break.

Reply:
Thanks for the suggestion, totally agree :-)